components/Slider/src/MDCSlider.m

```diff
@@ -478,6 +478,24 @@ - (void)accessibilityDecrement {
   }
 }
 
+- (BOOL)accessibilityActivate {
+  CGFloat midPoint = (self.maximumValue - self.minimumValue) / 2.0;
+  CGFloat newValue;
+  CGFloat adjustmentAmount = fabs((self.value - midPoint) / 3.0);
```

Comment:
Should we set some minimum adjustment amount? It seems that UISlider won't change when it's less than ~1.5%